training/model_management/src/azureml/model/mgmt/processors/transformers/convert.py

```diff
@@ -78,9 +78,8 @@ def _add_mlflow_signature(mlflow_model_path: Path, signature):
         yaml.dump(updated_yaml_dict, f)
 
 
-def _get_image_model_to_save(input_dir: Path, output_dir: Path, hf_conf: Dict = {}) -> Dict:
+def _get_image_model_to_save(input_dir: Path, output_dir: Path, temp_output_dir: Path, hf_conf: Dict = {}) -> Dict:
     """Save Huggingface image models to mlflow and return hftransformers accepted parameters."""
-    temp_output_dir = Path(output_dir).parent.absolute() / "tmp"
```

Comment:
Should we define "tmp" as some global variable and reuse it ?

Reply:
Done